com.syntevo.SmartGit.json

```diff
@@ -0,0 +1,85 @@
+{
+  "app-id": "com.syntevo.SmartGit",
+  "command": "smartgit",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "separate-locales": false,
+  "sdk": "org.gnome.Sdk",
+  "finish-args": [
+    "--device=dri",
+    "--env=SMARTGIT_JAVA_HOME=/app/extra/smartgit/jre",
+    "--filesystem=host",
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "smartgit",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 apply_extra /app/bin/apply_extra",
+        "install -Dm0755 smartgit.sh /app/bin/smartgit",
+        "install -Dm0644 smartgit-32.png /app/share/icons/hicolor/32x32/apps/com.syntevo.SmartGit.png",
+        "install -Dm0644 smartgit-48.png /app/share/icons/hicolor/48x48/apps/com.syntevo.SmartGit.png",
+        "install -Dm0644 smartgit-64.png /app/share/icons/hicolor/64x64/apps/com.syntevo.SmartGit.png",
+        "install -Dm0644 smartgit-128.png /app/share/icons/hicolor/128x128/apps/com.syntevo.SmartGit.png",
+        "install -Dm0644 smartgit-256.png /app/share/icons/hicolor/256x256/apps/com.syntevo.SmartGit.png",
+        "install -Dm0644 -t /app/share/appdata/ com.syntevo.SmartGit.appdata.xml",
+        "install -Dm0644 -t /app/share/applications/ com.syntevo.SmartGit.desktop"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "com.syntevo.SmartGit.appdata.xml"
+        },
+        {
+          "type": "file",
+          "path": "com.syntevo.SmartGit.desktop"
+        },
+        {
+          "type": "file",
+          "path": "smartgit-32.png"
+        },
+        {
+          "type": "file",
+          "path": "smartgit-48.png"
+        },
+        {
+          "type": "file",
+          "path": "smartgit-64.png"
+        },
+        {
+          "type": "file",
+          "path": "smartgit-128.png"
+        },
+        {
+          "type": "file",
+          "path": "smartgit-256.png"
+        },
+        {
+          "type": "extra-data",
+          "filename": "smartgit.tar.gz",
+          "only-arches": ["x86_64"],
+          "size": 68044229,
+          "url": "https://www.syntevo.com/downloads/smartgit/smartgit-linux-18_1_4.tar.gz",
+          "sha256": "95912b0264be6be4557cc046156db65e495046fff2076bb143b89f573473fea1"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "tar -xzf smartgit.tar.gz",
+            "rm -f smartgit.tar.gz"
+          ],
+          "dest-filename": "apply_extra"
+        },
+        {
+          "type": "script",
```

Comment:
A symlink should be enough.

Reply:
Hey, I don't understand what you meant here :disappointed:

I checked some repositories just to see the trend and pretty much all of them have this scheme/pattern.